quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -117,17 +143,46 @@ public static boolean isSupported(PropertyMapper<?> mapper) {
         return mapper.getCategory().getSupportLevel().equals(ConfigSupportLevel.SUPPORTED);
     }
 
+    public static Optional<PropertyMapper<?>> getDisabledMapper(String property) {
+        if (property == null) return Optional.empty();
+
+        return getDisabledMappers()
```

Comment:
> like the propertymappers, should we consider storing these in a map that is keyed by all of the possible candidates (to, from, cli, env).

@shawkins Yep, we probably can. 

> Also do we want to proactively consider the case where this is not 1-1? Like how we already have multiple propertymappers for example defined off of kc.db. Or should we worry about it once it comes up later?

I assume you mean the case when property mappers use the `mapFrom` attribute and the defined property by that is disabled, right? Yep, we can consider it as well. I assume the property mapper which uses disabled `mapFrom` prop should be also disabled, as it's the parent of it, right? I can try to implement it, but I'd rather do it in a follow-up task. I'd like to provide the functionality ASAP and then we can improve some other use-cases. WDYT?

Reply:
> I assume you mean the case when property mappers use the mapFrom attribute and the defined property by that is disabled, right?

Yes. db, fips-mode, log, and proxy-headers all map to multiple quarkus properties - we're not talking about disabling any of those yet, so it's more of a future concern.

>  I can try to implement it, but I'd rather do it in a follow-up task. I'd like to provide the functionality ASAP and then we can improve some other use-cases. WDYT?

That sounds good. Perhaps designating one of them as the parent, or even restructing so that PropertyMappers have child will make this clearer.